ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/PullQueryStreamWriter.java

```diff
@@ -92,7 +91,7 @@ public class PullQueryStreamWriter implements StreamingOutput {
     } catch (Exception e) {
       throw new KsqlStatementException(
           e.getMessage() == null
-              ? "Server Error" + Arrays.toString(e.getStackTrace())
```

Comment:
I just happened to notice some of these. It's never a good idea to just blindly pass the whole stacktrace back in an API response.